specs/Alt + Tab Spec.md

```diff
@@ -0,0 +1,51 @@
+# **PowerToys Alt+Tab Spec**
+
+![MS PowerToys](https://hothardware.com/ContentImages/NewsItem/48038/content/Microsoft_PowerToys.jpg "PowerToys")
+## Users can quickly access and organize open programs and tabs.
+### Authors: Benjamin Leverette and Prudence Phillips
+### Spec Status: Draft
+> ## 1. Overview
+&nbsp; &nbsp; &nbsp; &nbsp; **1.1. Elevator Pitch / Narrative** 
```

Comment:
I would consider removing the indentation using &nbsp, it makes the markdown harder to read.